docs/extras/headers.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Headers
+---
+# Headers Extra
+
+This extra implements the [RFC-8288](https://tools.ietf.org/html/rfc8288) compilant http response headers (and other helpers) useful for API pagination.
+
+- It removes the need for an extra dependency
+- No need to learn yet another interface
+- It saves quite a lot of memory and CPU
+- It works with any type of collection and/or `pagy_*` constructor (even `pagy_countless`)
+- It offers more explicit flexibility and simplicity
+
+## Synopsis
+
+See [extras](../extras.md) for general usage info.
+
+In the `pagy.rb` initializer:
+
+```ruby
+require 'pagy/extras/headers'
+```
+
+In your controller action:
+
+```ruby
+# paginate as usual with any pagy_* backend constructor
+pagy, records = pagy(Product.all)
+pagy_headers_merge(pagy)
+render json: records
+```
+
+Or if you can reuse it (e.g. in a pure API app with very standard actions), you are encouraged to define a custom `pagy_render` method in your application controller like:
+
+```ruby
+def pagy_render(collection, vars={})
+  pagy, records = pagy(collection, vars) # any pagy_* backend constructor works
+  pagy_headers_merge(pagy)
+  render json: records
+end
+
+# and use it in your standard actions:
+pagy_render(Product.all)
+```
+
+## Files
+
+This extra is composed of 1 small file:
```

Comment:
this comment is kinda redundant

Reply:
true. I will remove the stupid comments from all the extras and leave only the links to the files. Thanks